File: core/src/com/nopalsoft/dragracer/handlers/RequestHandler.java
package com.nopalsoft.dragracer.handlers;

public interface RequestHandler {
    void showRater();

    void showInterstitial();

    void showFacebook();

    void shareOnFacebook(final String message);

    void shareOnTwitter(final String message);

    void removeAds();

    void showAdBanner();

    void hideAdBanner();

    void buy50millionCoins();

}

File: core/src/com/nopalsoft/dragracer/handlers/RequestHandler.kt
package com.nopalsoft.dragracer.handlers

interface RequestHandler {
    fun showRater()

    fun showInterstitial()

    fun showFacebook()

    fun shareOnFacebook(message: String?)

    fun shareOnTwitter(message: String?)

    fun removeAds()

    fun showAdBanner()

    fun hideAdBanner()

    fun buy50millionCoins()
}
